monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/DatastoreQueryController.java

```diff
@@ -58,12 +66,16 @@
     @Autowired
     TenantContextFactory _tenantContextFactory;
 
+    @Autowired
+    private ApplicationEventPublisher eventPublisher;
```

Comment:
Looks like this has been added to the class, but I can see that it is never used.